config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -0,0 +1,26 @@
+presubmits:
+  kubernetes/k8s.io:
+  - name: pull-k8sio-cip
+    decorate: true
+    skip_report: false
+    run_if_changed: ".*/manifest.yaml"
+    # It's OK to run things concurrently, because we're using "-dry-run".
+    max_concurrency: 10
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:8ce6fba6cc5873648dd8bb8f9c591be0959e07b78a0381c796df2b359f990908
```

Comment:
It would be nice if there was some way to trace this to a commit. This is why we use names more like `cip-20190328-deadbeef` also if we document where/how to build this image (at least for the trusted variant of this job)

Reply:
Currently I just build it locally and push, but I do agree that there needs to be some more official way to do things (including naming the image a certain way, promoting it, etc). These concerns will be addressed in the near future.

Reply:
I mean just a comment about where the source code to this image lives. If you die, how is someone going to create the next version of the image? 

is it https://github.com/kubernetes-sigs/k8s-container-image-promoter I guess?